pkg/parquet/reader.go

```diff
@@ -37,6 +37,7 @@ type (
 type Reader interface {
 	ReadDate(ctx context.Context, datetime time.Time, target interface{}) error
 	ReadDateAsync(ctx context.Context, datetime time.Time, target interface{}, callback ResultCallback) error
+	ReadFileIntoTarget(ctx context.Context, file string, target interface{}) error
```

Comment:
Not part of this PR, but we should refactor the naming here. File is technically incorrect, because it is an s3 key that we use.

Reply:
i created #600 for this

Reply:
thank you